src/ucs/stats/stats.c

```diff
@@ -718,6 +930,17 @@ void ucs_stats_init()
     ucs_stats_set_trigger();
     kh_init_inplace(ucs_stats_cls, &ucs_stats_context.cls);
 
+    /* Aggregate-sum initialize database */
+    ucs_array_init_dynamic(&ucs_stats_context.aggrgt_sum_db);
+
+    /* Aggregate-sum class id to name database initialize */
+    ucs_stats_context.aggrgt_cnt_names_db = NULL;
+
+    /* We start from 1, index == 0 is reserved */
```

Comment:
why is it reserved?

Reply:
It is reserved since we don't want to use class_id == 0 (zero indicates that we don't have this class counters in the aggregate-sum list).

Reply:
can we use -1 instead of 0 for this indication?